Formula/ns-3.rb

```diff
@@ -26,41 +30,36 @@ class Ns3 < Formula
     depends_on "gcc"
   end
 
-  # `gcc version 5.4.0 older than minimum supported version 7.0.0`
-  fails_with gcc: "5"
+  fails_with gcc: "7"
 
   resource "pybindgen" do
     url "https://files.pythonhosted.org/packages/e0/8e/de441f26282eb869ac987c9a291af7e3773d93ffdb8e4add664b392ea439/PyBindGen-0.22.1.tar.gz"
     sha256 "8c7f22391a49a84518f5a2ad06e3a5b1e839d10e34da7631519c8a28fcba3764"
   end
 
-  # Fix ../src/lte/model/fdmt-ff-mac-scheduler.cc:1044:16: error:
-  # variable 'bytesTxed' set but not used [-Werror,-Wunused-but-set-variable]
-  # TODO: remove in the next release.
-  patch do
-    url "https://gitlab.com/nsnam/ns-3-dev/-/commit/dbd49741fcd5938edec17eddcde251b5dee25d05.diff"
-    sha256 "28e92297cfe058cfb587527dc3cfdb8ac66b51aba672be29539b6544591e2f1e"
-  end
-
   def install
-    resource("pybindgen").stage buildpath/"pybindgen"
+    resource("pybindgen").stage buildpath/"../pybindgen"
 
-    system "./waf", "configure", "--prefix=#{prefix}",
-                                 "--build-profile=release",
+    system "./ns3", "configure", "--prefix=#{prefix}",
+                                 "-d", "release",
                                  "--disable-gtk",
-                                 "--with-python=#{which("python3")}",
-                                 "--with-pybindgen=#{buildpath}/pybindgen"
-    system "./waf", "--jobs=#{ENV.make_jobs}"
-    system "./waf", "install"
+                                 "--enable-python-bindings",
+                                 "--enable-mpi",
+                                 "--", "-DCMAKE_INSTALL_RPATH=#{rpath}"
+    system "./ns3", "build", "--jobs=#{ENV.make_jobs}"
+    system "./ns3", "install"
+
+    # Starting 3.36, bindings are no longer installed
+    (lib/"python3.10/site-packages").install Dir["build/bindings/python/*"]
```

Comment:
```suggestion
    site_packages = Language::Python.site_packages("python3.10")
    (prefix/site_packages).install (buildpath/"build/bindings/python").children
```
I don't think you're operating on only directories here, so let's not use `Dir`.